From: Cordell Bloor <cgmb@slerp.xyz>
Date: Fri, 24 Nov 2023 14:25:32 -0700
Subject: disable xnack query print

If the amdgpu driver is built without HSA_AMD_SVM enabled, then KFD will
lack xnack support. This is normal and expected behaviour, so there is
no need to print a message.

Forwarded: no
Bug-Debian: https://bugs.debian.org/1056170
---
 src/core/runtime/amd_topology.cpp | 1 -
 1 file changed, 1 deletion(-)

--- a/src/core/runtime/amd_topology.cpp
+++ b/src/core/runtime/amd_topology.cpp
@@ -97,7 +97,6 @@
   // preference
   status = hsaKmtGetXNACKMode((HSAint32*)&mode);
   if(status != HSAKMT_STATUS_SUCCESS) {
-    debug_print("KFD does not support xnack mode query.\nROCr must assume xnack is disabled.\n");
     return false;
   }
   return mode;
